docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -653,5 +653,5 @@ curl -u elastic:<password> --request POST \
 [[php-key-download-fail]]
 == Air-gapped {agent} upgrade can fail due to an inaccessible PGP key
 
-In versions 8.9 and above, an {agent} upgrade may fail when the upgrader can't access a PGP key required to verify the binary signature. For details and a workaround, refer to the <<known-issue-3375,PGP key download fails in an air-gapped environment>> known issue in the version 8.9.0 Release Notes or to the link:https://github.com/elastic/elastic-agent/blob/main/docs/pgp-workaround.md[workaround documentation] in the elastic-agent GitHub repository.
+In versions 8.9 and above, an {agent} upgrade may fail when the upgrader can't access a PGP key required to verify the binary signature. For details and a workaround, refer to the link:https://www.elastic.co/guide/en/fleet/current/release-notes-8.9.0.html#known-issue-3375[PGP key download fails in an air-gapped environment] known issue in the version 8.9.0 Release Notes or to the link:https://github.com/elastic/elastic-agent/blob/main/docs/pgp-workaround.md[workaround documentation] in the elastic-agent GitHub repository.
```

Comment:
Minor nit
```suggestion
In versions 8.9 and above, an {agent} upgrade may fail when the upgrader can't access a PGP key required to verify the binary signature. For details and a workaround, refer to the {fleet-guide}/release-notes-8.9.0.html#known-issue-3375[PGP key download fails in an air-gapped environment] known issue in the version 8.9.0 Release Notes or to the link:https://github.com/elastic/elastic-agent/blob/main/docs/pgp-workaround.md[workaround documentation] in the elastic-agent GitHub repository.
```

Reply:
@jrodewig `{fleet-guuide}` contains the `{branch}` attribute, but this particular link will work only in the 8.9 docs, since the 8.9 release notes aren't present in 8.10 or later. So I don't think we can use that attribute here.